packages/enketo-express/public/js/src/module/gui.js

```diff
@@ -202,12 +202,14 @@ function feedback(message, duration, heading = t('feedback.header')) {
  * Shows a modal alert dialog.
  * TODO: parameters should change to (content, options)
  *
- * @param { string } message - message to show
+ * @param {string} message - message to show
  * @param {string=} heading - heading to show
  * @param {string=} level - css class or normal (no styling) ('alert', 'info', 'warning', 'error', 'success')
  * @param {number=} duration - duration in secondsafter which dialog should self-destruct
+ * @param {string=} buttonText - text for the button (default: OK)
+ * @returns {Promise} resolves when the alert is closed
  */
-function alert(message, heading, level, duration) {
+function alert(message, heading, level, duration, buttonText) {
```

Comment:
nit: this would be more clear that it's the default :thinking: 
```suggestion
function alert(message, heading, level, duration, buttonText = t('alert.validationsuccess.heading')) {
```